data/wsl/create_unattend_iso.sh

```diff
@@ -0,0 +1,29 @@
+ #!/bin/bash
```

Comment:
Remove leading space:

```suggestion
#!/bin/bash
```

Reply:
Done!